### DISCUSSION TITLE
Error creating PR Reviews - setup problem or bug?

### DISCUSSION THREAD

**Discussion by hermann-p** [magit/forge#380] [Support] [answered]
I set up `forge` successfully for Github by following the documentations of `forge` and `ghub`, and I can push/pull/create pr/create comments etc. But when I want to use `github-review-forge-pr-at-point`, I get

    Got an error from the Github API (wrong-type-argument stringp nil)

I believe the error might be somewhere in the resolution of the Github user name, as the list of requested reviewers in `forge`s PR-view shows `nil` where my name was to be expected. Still, my name seems to appear correctly in all other write activities I did using `forge`. Is this some setup problem or should I report a bug?

Edit: I'm not sure if it is really `github-review-forge-pr-at-point`, I use `' c r` from magit, of course.

**Comment by tarsius** [comment, marked as answer]
Use `M-x toggle-debug-on-error` to get a backtrace. And then post that in github-review's issue tracker.

**Reply by hermann-p** [reply]
Thanks - I didn't know this was from another package.
For future readers: `github-review` does not share `forge`'s auth, it needs an additional `Jon Doe^github-review` entry.
